edgelet/hsm-sys/azure-iot-hsm-c/src/edge_hsm_client_store.c

```diff
@@ -94,6 +94,19 @@ static HSM_STATE_T g_hsm_state = HSM_STATE_UNPROVISIONED;
 static CRYPTO_STORE* g_crypto_store = NULL;
 static int g_store_ref_count = 0;
 
+//##############################################################################
+// Helpers
+//##############################################################################
+
+#define FREEIF(x) \
+    do { \
+        if ((x != NULL)) { \
```

Comment:
>((x != NULL)) [](start = 11, length = 13)

Why the double parens around the conditional?  Typically, the parenthesis would be around `x`, like `((x) != NULL)`

Reply:
Agreed. Will fix here and other places.

Reply:
Fixed.